style.css

```diff
@@ -0,0 +1,396 @@
+}
+
+.heading {
+  color: #121212;
+  font-family: nyt-cheltenham, georgia, "times new roman", times, serif;
+  font-style: italic;
+  font-size: 40px;
+  position: relative;
+  left: 0;
+  text-align: center;
+}
+
+.Smoking-Gun-heading {
+  display: block;
+  margin-right: 319px;
+}
+
+.main-figure {
+  margin-top: 75px;
+  display: flex;
+  flex-direction: column;
+}
+
+.main-image-container {
+  justify-content: center;
+}
+
+.main-image {
+  width: 72%;
+  height: 92%;
+  margin: 25px auto 5px 231px;
+}
+
+.main-image-caption {
+  color: #666;
+  margin: 0 140px 0 231px;
+}
+
+.about-main-author {
+  display: flex;
+  justify-content: space-around;
+  margin-left: 158px;
+}
+
+.about-main-author ul {
+  display: flex;
+  justify-content: space-evenly;
+  list-style: none;
+}
+
+#dates {
+  padding-top: 29px;
+  margin-left: -448px;
+}
+
+.social {
+  padding: 25px 40px 0 0;
+}
+
+.social :hover {
+  background-color: silver;
+  cursor: pointer;
+}
+
+.article {
+  padding: 37px 1px 8px 0;
+  margin-left: 447px;
+  font-size: 1.25rem;
+  line-height: 1.875rem;
+}
+
+.subscription {
+  color: rgb(50, 104, 145);
+  font-weight: bold;
+  text-align: center;
+}
+
+.text {
+  margin-left: 596px;
+}
+
+.inflation p {
+  width: 211px;
+  font-family: nyt-franklin, arial, helvetica, sans-serif;
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 300;
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+}
+
+.inflation > span {
+  display: block;
+  margin-top: 25px;
+  margin-left: 227px;
+}
+
+.sub-text {
+  margin: 20px 0 15px 384px;
+  color: #666;
+}
+
+#sub {
+  margin: 20px 0 0 0;
+  color: #666;
+}
+
+.img-txt {
+  position: absolute;
+  top: 18.0556%;
+  left: 0.2544%;
+  width: 173px;
+}
+
+.img {
+  height: 50%;
+  width: 80%;
+}
+
+h1 {
+  margin-bottom: 2px;
+  line-height: 1.5rem;
+  font-weight: 700;
+  font-style: normal;
+  font-family: nyt-cheltenham, georgia, times new roman, times, serif;
+  font-size: 1.25rem;
+  color: black;
+}
+
+.univ-text {
+  position: relative;
+  font-size: smaller;
+  font-size: 14px;
+  font-family: nyt-cheltenham, georgia, times new roman, times, serif;
+}
+
+.title {
+  font-size: smaller;
+  font-weight: bold;
+}
+
+.inflation-section {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  padding-left: 17%;
+  padding-right: 17%;
+}
+
+.read-comments-button {
+  height: 35px;
+  width: 320px;
+  padding: 4px 12px 8px;
+  background-color: rgb(86, 123, 149);
+  color: white;
+  border-radius: 3px;
+  cursor: pointer;
+  margin-top: 29px;
+}
+
+#vedio {
+  margin: 27px 0 0 441px;
+}
+
+#comment {
+  color: #666;
+}
+
+.comment {
+  color: #666;
+  margin: 27px 0 0 441px;
+}
+
+.img-sky {
+  height: 376px;
+  margin: 36px 0 0 441px;
+}
+
+.article-one {
+  color: #666;
+  font-size: 0.9rem;
+  line-height: 1rem;
+}
+
+i.fa {
+  display: inline-block;
+  border-radius: 60px;
+  padding: 0.5em 0.6em;
+}
+
+.caption {
+  /* margin: 20px 0 0 114px; */
```

Comment:
Please remove the commented code. It makes your file look untidy. If you need to review a previous version of your code, use git history.